src/components/Question.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="grid grid-flow-col grid-rows-3 gap-4 pt-2 mt-2 border-b border-gray-100">
+    <div class="row-span-3 m-3 text-xl text-center text-gray-200">
+      {{ question.score }}
+      <p class="text-xxs">votes</p>
+      {{ question.answer_count }}
+      <p class="text-xxs">answers</p>
+
+      <p class="text-xxs">{{ question.view_count }} views</p>
+    </div>
+
+    <div class="col-span-2 row-span-2 text-lg">
+      <router-link :to="`/questions/${question.id}`">{{ question.title }}</router-link>
+      <!-- eslint-disable -->
```

Comment:
In your case eslint will be disabled for the rest of the file, what is not ok. Better to disable just for next line and just for one rile:
`<!-- eslint-disable-next-line vue/no-v-html -->`
or 
`<!-- eslint-disable-next-line no-v-html -->`

Reply:
Fixed